### PR TITLE
BTA-1609 : Fixed the error pattern

### PR DESCRIPTION
**Pull request by nehakannaujia1**
BTA-1609 : Fixed the error pattern

### CODE REVIEW

File: app/assets/stylesheets/partials/_forms.scss [hunk 1]
```diff
@@ -155,3 +155,7 @@ override character count display
         display: block;
     }
 }
+
+.form-group > fieldset > legend > span > h1.heading-large {
+margin-top :0
+}
```
Comment (suyangyu) [inline app/assets/stylesheets/partials/_forms.scss]:
```suggestion
}

```

File: app/assets/stylesheets/partials/_forms.scss [hunk 2]
```diff
@@ -155,3 +155,7 @@ override character count display
         display: block;
     }
 }
+
+.form-group > fieldset > legend > span > h1.heading-large {
+margin-top :0
```
Comment (suyangyu) [inline app/assets/stylesheets/partials/_forms.scss]:
```suggestion
    margin-top :0
```

File: app/views/components/input_radio.scala.html [hunk 3]
```diff
@@ -26,22 +26,22 @@
...
             <span class="bold-small @if(legendClass.nonEmpty){@{legendClass.get} span-inline-block}">@Html(legend)</span>
 
-            <div class="@if(field.hasErrors){form-field--error}">
+            <div>
                 @if(hint.nonEmpty){
                 <span class="form-hint @{hintTextSpacing}">@Html(hint.get)</span>
                 }
                 @field.errors.map { error =>
-                <span class="error-notification" id="error-message-@{field.id}-input">@messages(error.message, error.args: _*)</span>
+                 <span class="error-message margin-bottom-small" id="error-message-@{field.id}-input">@messages(error.message, error.args: _*)</span>
```
Comment (suyangyu) [inline app/views/components/input_radio.scala.html]:
```suggestion
                    <span class="error-message margin-bottom-small" id="error-message-@{field.id}-input">@messages(error.message, error.args: _*)</span>
```

can we re-indent this file please

Comment (nehakannaujia1) [reply app/views/components/input_radio.scala.html]: yeah done.

File: app/assets/stylesheets/partials/_forms.scss (hunk 2 above)
Comment (nehakannaujia1) [reply app/assets/stylesheets/partials/_forms.scss]: Done.

File: app/assets/stylesheets/partials/_forms.scss (hunk 1 above)
Comment (nehakannaujia1) [reply app/assets/stylesheets/partials/_forms.scss]: Its just styling changes so we do not need any unit tests.

**Pull request merged**